models/budget.js

```diff
@@ -11,11 +11,11 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      Budget.belongsTo(models.User);
```

Comment:
We use these specially named association methods- belongsTo, hasOne, hasMany- to set up the associations between models in Sequelize.

Here, we say that a Budget BELONGS TO a User. The belongsTo should be specified in the model that has the foreign key column.

This will add more methods into the Budget model class for accessing the associated User object for a Budget object.